### DISCUSSION TITLE
[Feature] IOS App, allow user to choose to not upload entire history

### DISCUSSION THREAD

**Discussion by psyciknz** [immich-app/immich#10103] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

There should be a switch, or user option, where it can be set to not upload the entire library.  And to start from today.

Or, and option to scan the device and check for matching assets.  I suspect the first option is much easier to implement.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4169
